Makefile

```diff
@@ -21,3 +21,7 @@ clean:
 
 app:
 	@$(REBAR) -r create template=mochiwebapp dest=$(DEST) appid=$(PROJECT)
+
+bench: app
```

Comment:
Not really relevant, I removed the Makefile changes altogether and just added bench.sh to the application skeleton. It didn't really make much sense where it was.